src/templates/layouts/vamcSystem/index.tsx

```diff
@@ -0,0 +1,175 @@
+import { ContentFooter } from '@/templates/common/contentFooter'
+import { LovellStaticPropsResource } from '@/lib/drupal/lovell/types'
+import { useEffect } from 'react'
+import { SideNavMenu } from '@/types/formatted/sideNav'
+// import { LovellSwitcher } from '@/templates/components/lovellSwitcher'
+// import { TopTasks } from '@/templates/components/topTasks'
+// import { FacilityListing } from '@/templates/components/facilityListing'
+// import { MainButtons } from '@/templates/components/mainButtons'
+// import { ListOfLinkTeasers } from '@/templates/components/listOfLinkTeasers'
+// import { NewsStoryTeaser } from '@/templates/components/newsStoryTeaser'
+// import { EventTeaser } from '@/templates/components/eventTeaser'
+// import { SocialLinks } from '@/templates/common/socialLinks'
+
+// Allows additions to window object without overwriting global type
+interface customWindow extends Window {
+  sideNav?: SideNavMenu
+}
+declare const window: customWindow
+
+export function VamcSystem({
+  title,
+  introText,
+  image,
+  administration,
+  fieldRelatedLinks,
+  path,
+  menu,
+  // vamcEhrSystem,
+  // mainFacilities,
+  // newsStoryTeasersFeatured,
+  // eventTeasersFeatured,
+  // eventTeasersAll,
+  // lovellVariant,
+  // lovellSwitchPath,
+}: LovellStaticPropsResource<FormattedVamcSystem>) {
+  // Populate the side nav data for the side nav widget to fill in
+  // Note: The side nav widget is in a separate app in the static-pages bundle
+  useEffect(() => {
+    window.sideNav = menu
+  })
+
+  // TODO: It looks like the reason the image is taller than in the original page is
+  // because the image file is actually different. It has a taller intrinsic height.
+  // Here is an example from `/washington-dc-health-care/`:
+  // Old:
+  // https://s3-us-gov-west-1.amazonaws.com/content.www.va.gov/img/styles/7_2_medium_thumbnail/public/2021-08/Washington%20VA%20Medical%20Center.jpg
+  // New:
+  // https://dsva-vagov-staging-cms-files.s3.us-gov-west-1.amazonaws.com/styles/2_1_large/public/2021-08/Washington%20VA%20Medical%20Center.jpg
+  const hasValidImage = image?.links?.['2_1_large']?.href
+
+  return (
+    <div className="va-l-detail-page va-facility-page">
+      <div className="usa-grid usa-grid-full">
+        {/* Nav data fille in by a separate script from `window.sideNav` */}
+        <nav aria-label="secondary" data-widget-type="side-nav" />
+        {/* Main page content */}
+        <div className="usa-width-three-fourths">
+          <article className="usa-content va-l-facility-detail vads-u-padding-bottom--0">
+            {/* <LovellSwitcher
+              currentVariant={lovellVariant}
+              switchPath={lovellSwitchPath}
+            /> */}
+            {title && <h1>{title}</h1>}
+            {hasValidImage && (
+              <div className="duotone darken lighten medium-screen:vads-u-margin-bottom--0p5">
+                <MediaImage {...image} imageStyle="2_1_large" />
```

Comment:
Does it need to be `7_2_medium_thumbnail`?
```suggestion
                <MediaImage {...image} imageStyle="7_2_medium_thumbnail" />
```

Reply:
That was it, thank you